data/yaml/venues/knowledgenlp.yaml

```diff
@@ -0,0 +1,2 @@
+acronym: KnowledgeNLP
```

Comment:
If this is the third, can you make sure that we don't have this venue already?

Reply:
Here is their website: https://knowledge-nlp.github.io/acl2024/

It appears they were not previously associated with ACL conferences.

Reply:
@mjpost Yes I also checked. So I'm assuming it's okay to have this workshop added as a new venue?

Reply:
Yes it seems to check out.